portality/static/js/edges/admin.background_jobs.edge.js

```diff
@@ -185,6 +186,20 @@ $.extend(true, doaj, {
                         hideInactive: true
                     })
                 }),
+                edges.newRefiningANDTermSelector({
+                    id: "queue_type",
+                    category: "facet",
+                    field: "queue_type.exact",
+                    display: "Queue Type",
+                    deactivateThreshold : 1,
+                    renderer: edges.bs3.newRefiningANDTermSelectorRenderer({
+                        controls: true,
+                        open: false,
+                        togglable: true,
+                        countFormat: countFormat,
+                        hideInactive: true
+                    })
+                }),
```

Comment:
Just one thing to add.  In the `SelectedFilters` component, the `queue_type.exact` fields needs to be mapped to a suitable human readable text.  Probably just "Queue Type" as we have here in the facet

Reply:
You can see the effect by selecting a value from the Queue type facet 
![Screenshot from 2022-11-05 09-50-38](https://user-images.githubusercontent.com/856138/200113880-8060eaa6-0019-4461-aaaf-8254b035cc52.png)